CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Delta Chat Android Changelog
 
+## v1.50.1
+2024-12
+
+* update to core 1.151.5
```

Comment:
I don't know if makes sense to mention any of the core changes, suggestions welcome

Reply:
"Encrypt notification tokens" from #3471 is added in core  1.151.5 and should maybe better moved here